jen/group.go

```diff
@@ -116,8 +116,13 @@ func (g *Group) renderItems(f *File, w io.Writer) (isNull bool, err error) {
 }
 
 // Render renders the Group to the provided writer.
-func (g *Group) Render(writer io.Writer) error {
-	f := NewFile("")
+func (g *Group) Render(writer io.Writer, file ...*File) error {
```

Comment:
It's generally though of as bad practise to use variadic parameters where there's an optional parameter, so `Render` should only the writer.

Reply:
Fixed, Thanks

Reply:
Yea, fixed